<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtYOLO</class>
 <widget class="QMainWindow" name="QtYOLO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtYOLO</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="slectFolder">
          <property name="text">
           <string>选择文件</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="modelBtn">
          <property name="text">
           <string>选择模型</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="Objdet">
          <property name="text">
           <string>目标检测</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="SegBtn">
          <property name="text">
           <string>实例分割</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="PoseBtn">
          <property name="text">
           <string>关键点检测</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="ObbBtn">
          <property name="text">
           <string>旋转框检测</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>490</width>
             <height>68</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>68</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>68</height>
            </size>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>123</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="tabStopDistance">
           <double>80.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
